webapp/tests/forms/steps/lotse/test_personal_data_steps.py

```diff
@@ -573,3 +577,79 @@ def test_if_required_value_is_not_give_validation_should_be_failure(self, new_te
                 StepPersonBHasDisability.name, True, ImmutableMultiDict(data))
             form = step.render_info.form
             assert form.validate() is False
+
+class TestPauschBetragPersonAValidation:
+    def test_if_required_value_is_given_then_validation_should_be_success(self, new_test_request_context):
+        data = MultiDict({'person_a_has_disability':'yes', 'person_a_requests_pauschbetrag': 'no'})
```

Comment:
```suggestion
        data = MultiDict({'person_a_has_disability':'yes'})
```
Same as below I think?

Reply:
No, person_a_requests_pauschbetrag is not opional